studio/components/interfaces/Auth/Policies/PolicySelection.tsx

```diff
@@ -58,18 +60,23 @@ const PolicySelection = ({
             onClick={onViewEditor}
           />
         </div>
-        <p className="text-sm text-foreground-light">
-          Not sure what policies are? Check out our resources{' '}
-          <a
-            target="_blank"
-            rel="noreferrer"
-            className="text-brand transition-colors hover:text-brand-600"
-            href="https://supabase.com/docs/guides/auth#policies"
-          >
-            here
-          </a>
-          .
-        </p>
+        <div>
+          <p className="text-sm text-foreground-light">
+            Not sure what policies are? Check out our resources{' '}
+            <a
+              target="_blank"
+              rel="noreferrer"
+              className="text-brand transition-colors hover:text-brand-600"
+              href="https://supabase.com/docs/guides/auth#policies"
+            >
+              here
+            </a>
+            .
+          </p>
+          <div className="flex justify-end w-full items-center space-x-3" onClick={onSelectCancel}>
+            <Button type="default">Cancel</Button>
```

Comment:
move onClick to `<Button>`

Reply:
Hi, thanks for reviewing the code, overlooked on this. Have amended the code!